### PR TITLE
 Close #15383 : New format for funnel of prospection 

### LINKED ISSUE #15383: New widget: Tunnel of prospection
Add a widget with a graphic per status of prospection (similar to graphic into home page of project).

### PR DESCRIPTION
**Pull request by Hystepik**

### CODE REVIEW

File: htdocs/core/boxes/box_funnel_of_prospection.php [hunk 1]
```diff
@@ -97,6 +97,8 @@ public function loadBox($max = 5)
 		$sql = "SELECT cls.rowid, cls.code, cls.percent, cls.label";
 		$sql .= " FROM " . MAIN_DB_PREFIX . "c_lead_status as cls";
 		$sql .= " WHERE active=1";
+		$sql .= " Order by cls.rowid";
```
Comment (eldy) [inline htdocs/core/boxes/box_funnel_of_prospection.php:100]: For this use instead   $this->db->order(...

File: htdocs/core/boxes/box_funnel_of_prospection.php [hunk 2]
```diff
@@ -97,6 +97,8 @@ public function loadBox($max = 5)
...
+		$sql .= " AND cls.code != 'LOST'";
```
Comment (eldy) [inline htdocs/core/boxes/box_funnel_of_prospection.php:101]: the different operator for SQL is <>   ( !=  is supported but not standard)

File: htdocs/core/boxes/box_funnel_of_prospection.php (hunk 1 above)
Comment (Hystepik) [reply htdocs/core/boxes/box_funnel_of_prospection.php]: it's fixed

File: htdocs/core/boxes/box_funnel_of_prospection.php (hunk 2 above)
Comment (Hystepik) [reply htdocs/core/boxes/box_funnel_of_prospection.php]: it's fixed

**Comment by Hystepik** [comment]
The new format of the funnel of prospection
image

**Pull request merged**